### PR TITLE
Add miniature golf preset

### PR DESCRIPTION
**Pull request by boothym**
Not sure why it's also submitting my previous edit in this pull request - no idea how to get rid of it.

### CODE REVIEW

File: data/presets/presets/leisure/miniature_golf.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "operator",
+        "address",
+        "opening_hours"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "crazy"
```
Comment (bhousel) [inline data/presets/presets/leisure/miniature_golf.json]: This needs commas in the terms array..

**Comment by bhousel** [comment]
> Not sure why it's also submitting my previous edit in this pull request - no idea how to get rid of it.

That's because your master branch is behind the main master branch.. When you make a branch, the "where it branched off from" pointer never changes unless you rebase the branch.

Github is just trying to collect all the things in your master branch that have changed since that branch was created.  So a good workflow to avoid this is:

Set your remotes up like this
```
git remote set origin git@github.com:boothym/iD.git           # probably already there
git remote add upstream git@github.com:openstreetmap/iD.git
```

Then, sometimes you'll want to keep your master in sync with ours (to avoid the problem you're seeing):
```
git checkout master                # make sure you are on the master branch to start
git fetch --all                    # so git knows what is new
git reset --hard upstream/master   # makes your master exactly like upstream master
```

Then when you want to work on something:
```
git checkout -b <work-branch>      # start a new branch just for the work you're doing
... do work...
git push origin                    # pushes work-branch to your github
... go to github and open pull request from work-branch back to upstream/master...
```

Some more info here:  http://stackoverflow.com/questions/16329776/how-to-keep-a-git-branch-in-sync-with-master

**Comment by bhousel** [comment]
Anyway, everything else about your PR was totally great except for the missing commas, so I just merged it as:

ad11770 2016-07-27 Create miniature_golf.json <boothym>
5de925b 2016-07-27 Fix commas, run make <Bryan Housel>

Let me know if you have any other questions, happy to help people work with git..

**Comment by boothym** [comment]
Whoops, thanks for fixing the commas! I don't actually have git installed (yet) so I've just used the website to make the edits. Currently trying to find out how to keep the fork in sync via the web UI.

**Pull request closed without merging**